pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -57,13 +59,19 @@ const (
 
 	// LocalhostAddress for local binding
 	LocalhostAddress = "127.0.0.1"
+
+	traceSamplingDefault = 10.0
```

Comment:
Btw this will also cause a lot of UX issues for people playing around with tracing or starting off in Istio. The 10%  should be something we recommend in docs, in Operations Guide. For people who are demoing istio or playing with it, we need to keep it at 100% to avoid the element of surprise.

Reply:
Good point, I'll change it.

Reply:
Done